crates/sui-benchmark/src/options.rs

```diff
@@ -40,6 +40,8 @@ pub struct Opts {
     /// with large enough gas i.e. u64::MAX
     #[clap(long, default_value = "", global = true)]
     pub primary_gas_id: String,
+    #[clap(long, default_value = "", global = true)]
+    pub pay_coin_id: String,
```

Comment:
@longbowlu - should we follow the same approach of passing in an object id and offset for pay coin (like primary gas) as well? This is mainly because we can use the same stress binary across all deploys (current private testnet and devnet, and wave 2). So, for current deployments we will pass the same gas id for both primary gas and pay coin and with an offset of 5000, they will pick a different random object ids. For wave 2, we will pass a different primary gas and pay coin id with an offset of 1. Lmk or I can do that change if we just want to push this through

Reply:
this is a hot fix to unblock things and I don't intent to merge it :p (although the caveat is we may need to rebuild stress again if there are new changes to wave2 core protocol) But your suggestion makes sense, I will look once your PR is out